recipes/crashpad/all/build.py

```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-import os
-import platform
-from bincrafters import build_template_default
-
-if __name__ == "__main__":
-
-    if platform.system() == "Linux":
-        command = os.environ["CONAN_DOCKER_ENTRY_SCRIPT"] + "; sudo apt-get -qq update && sudo apt-get -qq install -y python-minimal"
```

Comment:
Is there a way to set env var `CONAN_DOCKER_ENTRY_SCRIPT` somewhere else ?

Reply:
No, I don't think there is any way to define an environment variable for CI tooling for a single recipe